modules/node/src/appRegistry/appRegistry.service.ts

```diff
@@ -333,8 +336,23 @@ export class AppRegistryService implements OnModuleInit {
     const channel = await this.channelRepository.findByMultisigAddressOrThrow(
       appInstance.multisigAddress,
     );
-    if (channel.activeCollateralizations[latestState.assetId]) {
-      throw new Error(`Cannot uninstall deposit app with active collateralization`);
+    const depositApps = await this.cfCoreService.getAppInstancesByAppDefinition(
```

Comment:
Could be worth extracting this into a helper function.

Reply:
thought about it, but its really only used twice so i just left it